modules/@ergonode/products/src/components/Grids/ProductsGrid.vue

```diff
@@ -233,7 +225,12 @@ export default {
             'disabledElements',
         ]),
         extendedActionHeader() {
-            return this.$getExtendSlot('@Products/components/Grids/ProductsGrid/actionHeader');
+            const extendedActionHeader = this.$getExtendSlot('@Products/components/Grids/ProductsGrid/actionHeader');
+
+            if (!extendedActionHeader || !extendedActionHeader.__ALL) {
```

Comment:
The `__ALL` key is used mostly in forms where it is known that we have a type separation. In this case, there will be no different types, in addition, you impose that you return only the key `__ALL`. maybe it's better to use an array instead of an object.